deploy/helm_charts/envs/unsdg_staging.yaml

```diff
@@ -42,17 +43,22 @@ serviceGroups:
   svg:
```

Comment:
can remove svg here?

Reply:
or do you just want the default mixer service only?

Reply:
Removed the extra service groups & deployed to staging.unsdg.datacommons.org